ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/models/DestinationList.java

```diff
@@ -0,0 +1,67 @@
+package ca.ualberta.cs.shinyexpensetracker.models;
+
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DestinationList extends Model<DestinationList> {
```

Comment:
That stuff was pulled from Ramish's branch (#18), which I believe he was in the middle of fixing when I pulled.